src/seedsigner/views/screensaver.py

```diff
@@ -183,50 +187,58 @@ def start(self):
         # Store the current screen in order to restore it later
         self.last_screen = self.renderer.canvas.copy()
 
-        screensaver_start = int(time.time() * 1000)
-
         # Screensaver must block any attempts to use the Renderer in another thread so it
         # never gives up the lock until it returns.
         with self.renderer.lock:
             try:
+                last_update_time = time.time()
+                current_speed = self.rand_speed() * self.pixels_per_second
```

Comment:
Why break the speed calc into two pieces (rand * constant)?

The `rand_speed()` function could just be built to range from some min_speed (pixels per sec) to max_speed. It would make it much clearer what the actual min/max speed is (constant * 1.3 is much less intuitive).

Also for future compatibility: we'll eventually have screens with higher pixel counts. So I would calculate the min/max speed based on the smallest dimension (e.g. 240 for a 240x320 screen). Can eyeball results to come up with an opinionated min/max speed, but something like max speed is: 1.0s to cross the min dimension (so 240px/s in this case but 320px/s in some future 320x480 screen).